zproject/default_settings.py

```diff
@@ -494,6 +494,18 @@
 # above which sending of presence update events will be disabled.
 USER_LIMIT_FOR_SENDING_PRESENCE_UPDATE_EVENTS = 100
 
+# Controls the how much newer a user presence update needs to be
+# than the currently saved last_active_time in order for us to
```

Comment:
I think the reference to `last_active_time` is no longer accurate? Maybe "the currently saved last_active_time or last_connected_time..." would work?